go/vt/sqlparser/ast_format.go

```diff
@@ -2754,3 +2754,9 @@ func (node *Variable) Format(buf *TrackedBuffer) {
 	}
 	buf.astPrintf(node, "%v", node.Name)
 }
+
+// Format formats the node.
+func (node *PointExpr) Format(buf *TrackedBuffer) {
+	buf.WriteString("point")
+	buf.astPrintf(node, "(%v, %v)", node.XCordinate, node.YCordinate)
```

Comment:
Why don't you merge the first call into the second one itself like - 
```go
buf.astPrintf(node, "point (%v, %v)", node.XCordinate, node.YCordinate)
```

Reply:
@GuptaManan100 How would it case it in the canonical form with this style? 

@Ayman161803 I think it's good to also add a canonical format test to the `tracked_buffer_test.go` for a `point`. So we know what it looks like / what we want it to look like there as well.

Reply:
@dbussink I have added test cases assuming ``POINT(x, y)`` to be the right canonical form.